game/views.py

```
@@ -760,10 +790,10 @@ def verify_otp(request):
                     )
                     email.attach_alternative(html_content, "text/html")
                     email.send(fail_silently=True)
-                
+
                 except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)